# HG changeset patch
# User jwe
# Date 745020860 0
#      Tue Aug 10 22:14:20 1993 +0000
# Node ID 82c3febbaf04eef51317ea696e9088613f4e9c2b
# Parent  cd5ac1c52ea605138a3bfd1be28e7e39ac106821
[project @ 1993-08-10 22:14:20 by jwe]
Add libcruft/balgen/Makefile to the list of files to frob.

diff --git a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -378,18 +378,18 @@ fi
 
 AC_PROGRAM_CHECK(RUNTEST, runtest, runtest, [])dnl
 AC_PROGRAM_CHECK(EXPECT, expect, expect, [])dnl
 if test -z "$EXPECT" ; then
   RUNTEST=
 fi
 AC_SUBST(RUNTEST)dnl
 
-define([tmpa], [libcruft/blas/Makefile libcruft/dassl/Makefile])dnl
-define([tmpb], [libcruft/fftpack/Makefile])dnl
+define([tmpa], [libcruft/blas/Makefile libcruft/balgen/Makefile])dnl
+define([tmpb], [libcruft/dassl/Makefile libcruft/fftpack/Makefile])dnl
 define([tmpc], [libcruft/fsqp/Makefile libcruft/lapack/Makefile])dnl
 define([tmpd], [libcruft/linpack/Makefile libcruft/minpack/Makefile])dnl
 define([tmpe], [libcruft/misc/Makefile libcruft/npsol/Makefile])dnl
 define([tmpf], [libcruft/odepack/Makefile libcruft/qpsol/Makefile])dnl
 define([tmpg], [libcruft/quadpack/Makefile libcruft/ranlib/Makefile])dnl
 define([tmph], [libcruft/villad/Makefile])dnl 
 define([cruftdirs], [tmpa tmpb tmpc tmpd tmpe tmpf tmpg tmph])dnl
 define([tmpA], [Makefile Makeconf libcruft/Makefile libcruft/Makerules])dnl
